src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -78,12 +78,24 @@ public JwtHeader(EncryptingCredentials encryptingCredentials)
         /// <param name="signingCredentials"><see cref="SigningCredentials"/> used when creating a JWS Compact JSON.</param>
         /// <param name="outboundAlgorithmMap">provides a mapping for the 'alg' value so that values are within the JWT namespace.</param>
         public JwtHeader(SigningCredentials signingCredentials, IDictionary<string,string> outboundAlgorithmMap)
+            : this(signingCredentials, outboundAlgorithmMap, null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="JwtHeader"/>.
+        /// With the Header Parameters:
+        /// <para>{ { typ, JWT }, { alg, SigningCredentials.Algorithm } }</para>
+        /// </summary>
+        /// <param name="signingCredentials"><see cref="SigningCredentials"/> used when creating a JWS Compact JSON.</param>
+        /// <param name="outboundAlgorithmMap">provides a mapping for the 'alg' value so that values are within the JWT namespace.</param>
+        /// <param name="tokenType"> provides the token type</param>
```

Comment:
nit: extra blank space at the beginning.
Perhaps we could clarify this comment a bit, saying which claim will be populated and what happens if null or empty string is passed.

Reply:
Fixed it.